### PR TITLE
Update rpm.md

### PR DESCRIPTION
**Pull request by williamtrelawny**
Fixed code blocking

### Description
_Describe what this change achieves._

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks so much for catching and fixing this, @williamtrelawny! Looks good.

**Comment by hdhalter** [comment]
@Naarcha-AWS - Since you authored this piece, can you please confirm that it looks good and then merge? Thanks! 
image

File: _install-and-configure/install-opensearch/rpm.md [hunk 1]
```diff
@@ -42,7 +42,20 @@ This guide assumes that you are comfortable working from the Linux command line
     sudo rpm --import https://artifacts.opensearch.org/publickeys/opensearch.pgp
     ```
     {% include copy.html %}
-
+1. For OpenSearch 2.12 and greater, a custom admin password is required in order to set up a security demo configuration.  To set a custom admin password, use one the following commands:
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/rpm.md:45]:
```suggestion

1. For OpenSearch 2.12 and greater, a custom admin password is required in order to set up a security demo configuration.  To set a custom admin password, use one the following commands:
```

File: _install-and-configure/install-opensearch/rpm.md [hunk 2]
```diff
@@ -57,36 +70,23 @@ This guide assumes that you are comfortable working from the Linux command line
...
-    ```bash
-    sudo systemctl start opensearch
-    ```
-    {% include copy.html %}
+   ```bash
+   sudo systemctl start opensearch
+   ```
+   {% include copy.html %}
 
 1. Verify that OpenSearch launched correctly.
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/rpm.md:85]: Beyond the line break, what status should users expect OpenSearch to return when running this command?

```suggestion
1. Verify that OpenSearch launched correctly:

```

File: _install-and-configure/install-opensearch/rpm.md [hunk 3]
```diff
@@ -57,36 +70,23 @@ This guide assumes that you are comfortable working from the Linux command line
...
-
-   # Install the x64 package using rpm.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
-
-   # Install the arm64 package using yum.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> yum install opensearch-{{site.opensearch_version}}-linux-x64.rpm
-
-   # Install the arm64 package using rpm.
-   sudo env OPENSEARCH_INITIAL_ADMIN_PASSWORD=<custom-admin-password> rpm -ivh opensearch-{{site.opensearch_version}}-linux-x64.rpm
 1. After the installation succeeds, enable OpenSearch as a service.
```
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/rpm.md:73]:
```suggestion
1. After the installation succeeds, enable OpenSearch as a service:

```

**Review by Naarcha-AWS — CHANGES_REQUESTED** [review]
One follow up questions and suggested adding line breaks between the code examples.

File: _install-and-configure/install-opensearch/rpm.md (hunk 1 above)
Comment (hdhalter) [inline _install-and-configure/install-opensearch/rpm.md:45]:
```suggestion
1. For OpenSearch 2.12 and later, a custom admin password is required in order to set up a security demo configuration. To set a custom admin password, use one the following commands:
```

File: _install-and-configure/install-opensearch/rpm.md (hunk 2 above)
Comment (williamtrelawny) [reply _install-and-configure/install-opensearch/rpm.md:85]: If users run `curl -I localhost:9200` and get `HTTP/1.1 200 OK` in the response, this should confirm all is well.

**Review by Naarcha-AWS — APPROVED** [review]

File: _install-and-configure/install-opensearch/rpm.md (hunk 1 above)
Comment (Naarcha-AWS) [inline _install-and-configure/install-opensearch/rpm.md:45]:
```suggestion
1. For OpenSearch 2.12 and greater, a custom admin password is required in order to set up a security demo configuration. To set a custom admin password, use one the following commands:
```

**Pull request merged**